.github/workflows/lint.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
     steps:
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: 3.6
```

Comment:
Python min-ver is 3.7, so yes, that can be updated.

Macos wheels are 3.8 and up, because that's the first version that support 'fat wheels'. People who use 3.7 should still be able to build themselves.